KsApi/models/graphql/adapters/Project+FetchProjectQueryData.swift

```diff
@@ -114,4 +121,28 @@ extension Project {
 
     return (project, projectBackingId)
   }
+
+  /** FIXME: This is unfortunately a consequence of the no-reward reward being returned on v1 but not in GQL. Eventually we'll want to talk with backend about the possibility of returning a no-reward reward as part of the project query, just as they did with v1. The benefit of that is no reward reward doesn't have to be maintained locally. We want to show the rewards that the backend returns without modification to the raw data.
+   */
+
+  private static func noRewardReward(from fragment: GraphAPI.ProjectFragment?) -> Reward {
+    var projectMinimumPledgeAmount = 1.0
```

Comment:
Could 129-138 be simplified with a nil-coalescing operator on one line for each of these variables?

Reply:
Also curious if the fxRate ever returns as nil? Seems like something that we should never be hard-coding a value for.

Reply:
Good call with the `nil-coelesing`, added that. 
The `fxRate` and `minPledge` always have a value from GQL, never `null`, but I guess because we pass the project fragment here as an optional, there has to be some unwrapping. I could pass in an non-optional project fragment and wrap this around an `if-let` at the call-site if that's preffered.

setting the `fxRate` to 1 just means we return whatever the minimum pledge amount is for the reward in the currency of the project.